### PR TITLE
Mobile View

### PR DESCRIPTION
**Pull request by pinnockf**
image

### CODE REVIEW

File: jtflask/static/assets/js/splash.js [hunk 1]
```diff
@@ -1,29 +1,42 @@
...
+      );
+      setTimeout(() => {
+        splashDiv.setAttribute("hidden", "true");
+      }, 3000);
+    });
+
+    splashDiv.addEventListener("click", () => {
+      splashDiv.setAttribute("hidden", "true");
+    });
+  };
```
Comment (pinnockf) [inline jtflask/static/assets/js/splash.js:37]: Click splash screen to skip animation

File: jtflask/static/assets/css/navbar.css
Comment (pinnockf) [inline jtflask/static/assets/css/navbar.css:1]: Renamed sidebar.css to navbar.css to correspond with Appshell.navbar

File: jtflask/jtdash/layout.py [hunk 2]
```diff
@@ -53,6 +52,7 @@
                 <img class='logo-image' src='/jtdash/assets/images/splash-logo-image.png'/>
             </div>
         </div>
+        <div id='digital-twin-container'></div>
         {%app_entry%}
```
Comment (pinnockf) [inline jtflask/jtdash/layout.py:56]: Moved digital twin container to be a sibling to the rest of the app. To fix z-index issues

**Comment by chjch** [comment]
migrated all changes to the main branch. Close PR.

**Pull request closed without merging**